internal/provisioner/plain/controllers/bundleinstance_controller.go

```diff
@@ -263,9 +263,10 @@ func (r *BundleInstanceReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		}
 	}
 	meta.SetStatusCondition(&bi.Status.Conditions, metav1.Condition{
-		Type:   rukpakv1alpha1.TypeInstalled,
-		Status: metav1.ConditionTrue,
-		Reason: rukpakv1alpha1.ReasonInstallationSucceeded,
+		Type:    rukpakv1alpha1.TypeInstalled,
+		Status:  metav1.ConditionTrue,
+		Reason:  rukpakv1alpha1.ReasonInstallationSucceeded,
+		Message: "installation succeeded",
```

Comment:
WDYT about making this a bit more helpful and including the name of the bundle that was installed as part of the message? Also, technically the Bundle is "installed" by the bundle controller, the BI controller is creating an instance of that Bundle on-cluster, so I wonder if we can be more precise with our language here. 

For example,
```
fmt.Sprintf(instantiated bundle %s successfully, bi.Spec.BundleName)
```